# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1224005169 14400
#      Tue Oct 14 13:26:09 2008 -0400
# Node ID 263261b2124fbc0f55ecf4b8bfcd347035bb9a37
# Parent  d5e08881bba88fa781db014c8fc8a30710e5ae90
ishghandle.m: remove spurious \n\ from doc string

diff --git a/scripts/plot/ishghandle.m b/scripts/plot/ishghandle.m
--- a/scripts/plot/ishghandle.m
+++ b/scripts/plot/ishghandle.m
@@ -13,16 +13,16 @@
 ## General Public License for more details.
 ##
 ## You should have received a copy of the GNU General Public License
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @deftypefn {Function File} {} ishghandle (@var{h})
-## Return true if @var{h} is a graphics handle and false otherwise.\n\
+## Return true if @var{h} is a graphics handle and false otherwise.
 ## @end deftypefn
 
 function retval = ishghandle (h)
   ## This function is just included for compatibility as Octave has 
   ## no simulink equivalent.
   retval = ishandle (h);
 endfunction
